timesync/clock.go

```diff
@@ -97,7 +173,7 @@ func (t *Ticker) Subscribe() LayerTimer {
 	t.m.Lock()
 	t.subscribers[ch] = struct{}{}
 	t.m.Unlock()
-
+	log.Info("subscribed to channel")
```

Comment:
this log has no context

Reply:
this is good if you want to count number of subscribers